terraform/environments/mlra/ecs.tf

```diff
@@ -33,79 +33,10 @@ module "mlra-ecs" {
   ecs_scaling_mem_threshold = local.application_data.accounts[local.environment].ecs_scaling_mem_threshold
 
-  depends_on = [module.alb, aws_cloudwatch_log_group.ecs_log_group] # TODO module.alb dependancy may have to be re-factored further into development
-}
-
-locals {
-  ec2_ingress_rules = {
-    "cluster_ec2_lb_ingress" = {
-      description     = "Cluster EC2 ingress rule"
-      from_port       = 22
-      to_port         = 22
-      protocol        = "tcp"
-      cidr_blocks     = [data.aws_vpc.shared.cidr_block]
-      security_groups = []
-    }
-    "cluster_ec2_lb_ingress_2" = {
-      description     = "Cluster EC2 ingress rule 2"
-      from_port       = 8080
-      to_port         = 8080
-      protocol        = "tcp"
-      cidr_blocks     = [data.aws_vpc.shared.cidr_block]
-      security_groups = []
-    }
-    "cluster_ec2_lb_ingress_3" = {
-      description     = "Cluster EC2 ingress rule 3"
-      from_port       = 32768
-      to_port         = 61000
-      protocol        = "tcp"
-      cidr_blocks     = [data.aws_vpc.shared.cidr_block]
-      security_groups = [module.alb.security_group.id]
-    }
-    "cluster_ec2_lb_ingress_4" = {
-      description     = "Cluster EC2 ingress rule 4"
-      from_port       = 1521
-      to_port         = 1521
-      protocol        = "tcp"
-      cidr_blocks     = [data.aws_vpc.shared.cidr_block]
-      security_groups = [module.alb.security_group.id]
-    }
-  }
-  ec2_egress_rules = {
-    "cluster_ec2_lb_egress" = {
-      description     = "Cluster EC2 loadbalancer egress rule"
-      from_port       = 0
-      to_port         = 0
-      protocol        = "-1"
-      cidr_blocks     = ["0.0.0.0/0"]
-      security_groups = []
-    }
-  }
-
-  user_data = base64encode(templatefile("user_data.sh", {
-    app_name = local.application_name
-  }))
-
-  task_definition = templatefile("task_definition.json", {
-    app_name            = local.application_name
-    ecr_url             = local.application_data.accounts[local.environment].ecr_url
-    docker_image_tag    = local.application_data.accounts[local.environment].docker_image_tag
-    region              = local.application_data.accounts[local.environment].region
-    maat_api_end_point  = local.application_data.accounts[local.environment].maat_api_end_point
-    maat_db_url         = local.application_data.accounts[local.environment].maat_db_url
-    maat_db_password    = data.aws_ssm_parameter.db_password.value
-    maat_libra_wsdl_url = local.application_data.accounts[local.environment].maat_libra_wsdl_url
-    sentry_env          = local.environment
-  })
 }
 
 # MAAT DB Password
 
 data "aws_ssm_parameter" "db_password" {
   name = local.application_data.accounts[local.environment].maat_db_password
 }
-
-#TODO This needs to be added in the cloudwatch module in the future
-resource "aws_cloudwatch_log_group" "ecs_log_group" {
-  name = "${local.application_name}-ecs-log-group"
-}
```

Comment:
This is not going into the cloudwatch module since it is tied with ECS. It is now defined within the ecs module